### DISCUSSION TITLE
Nice interview, @Splaxi

### DISCUSSION THREAD

**Discussion by FH-Inway** [d365collaborative/d365fo.tools#637] [General]
Really liked the interview @Splaxi did with @JoPe72 and [@g_sundblad](https://twitter.com/g_sundblad) from the Dynamics Update Podcast: https://dynamicsupdate.podbean.com/e/interview-motzjensen/

Nice bit of history for the d365fo.tools and some good points on testing, the CDE program and more :+1:

**Comment by Splaxi** [comment]
@FH-Inway Thank you for the kind words 🤘
